added open-rag-eval

ofermend opened the pull request:
Added for RAG evaluation with example notebook.

ofermend committed:
added open-rag-eval link and example notebook

NirDiamant commented:
Hi, thanks first of all!
Several points to enhance: 
- Please make the textual explanation at the beginning adhere to the repo's format (look at the CONTRIBUTING.MD file or at other tutorials for reference). 
- Add markdown descriptions above each cell
- give a better understanding of the use case data that you chose to show in this tutorial 
- Clean any unnecessary cell outputs
- Try to make this flow as if you are holding a 7-year-old reader's hand towards the tutorial. It should be crystal clear. 

Thanks!

ofermend committed:
updated per comments from Nir

ofermend commented:
Thanks for the review and comments.
I updated the notebook: added overview and cleaned up the cells. This notebook is just intended to show a "how to use" example as opposed to explain how it works internally, so I just added a link to our blog which has the details of how it works internally - would that be okay?

NirDiamant commented:
Closing due to extended inactivity. The repo has evolved significantly since this was submitted. If you'd like to contribute this, please open a fresh PR against the current main branch. Thank you for your interest in the project.

NirDiamant commented:
Content merged manually to resolve README conflicts. Thank you for the contribution - your notebook is now live on main.
